app/models/form.rb

```diff
@@ -41,6 +41,18 @@ def target_for_delivery_method
     errors.add(:element_selector, "can't be blank for an inline form") if (delivery_method == 'custom-button-modal' || delivery_method == 'inline') && (element_selector == '')
   end
 
+  def roles
+    user_roles.map { |role| { role: role.role, user: role.user }}
+  end
+
+  def form_managers
```

Comment:
could select from user_roles here too...  but tried to keep the change minimal (moving this from the view)